src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -56,6 +56,8 @@ type MoneyRequestViewProps = {
     /** Whether we should show Money Request with disabled all fields */
     readonly?: boolean;
 
+    isFromReviewDuplicates?: boolean;
```

Comment:
NAB but let's add comment (/** comment above it */) to the variable to be compliant with checklist.